output/cloud/output_test.go

```diff
@@ -868,28 +819,76 @@ func TestPublishMetric(t *testing.T) {
 		},
 	}
 	err = out.client.PushMetric("1", samples)
-
-	assert.Nil(t, err)
+	assert.NoError(t, err)
 }
 
 func TestNewOutputClientTimeout(t *testing.T) {
 	t.Parallel()
 	ts := httptest.NewServer(http.HandlerFunc(func(rw http.ResponseWriter, r *http.Request) {
-		time.Sleep(5 * time.Millisecond)
+		time.Sleep(100 * time.Millisecond)
 	}))
 	defer ts.Close()
 
 	out, err := newOutput(output.Params{
 		Logger:     testutils.NewLogger(t),
 		JSONConfig: json.RawMessage(fmt.Sprintf(`{"host": "%s",  "timeout": "2ms"}`, ts.URL)),
 		ScriptOptions: lib.Options{
-			Duration:   types.NullDurationFrom(1 * time.Second),
+			Duration:   types.NullDurationFrom(50 * time.Millisecond),
 			SystemTags: &metrics.DefaultSystemTagSet,
 		},
 		ScriptPath: &url.URL{Path: "script.js"},
 	})
 	require.NoError(t, err)
 
 	err = out.client.PushMetric("testmetric", nil)
-	assert.True(t, os.IsTimeout(err))
+
+	var nerr net.Error
+	require.True(t, errors.As(err, &nerr))
+	assert.True(t, nerr.Timeout())
+}
+
+func TestOutputStartStopTestCreationWithConfigOverwrite(t *testing.T) {
```

Comment:
I'm unsure why the test name contains start/stop.

Is it just something like: `TestOutputConfigOverwriting`?

Reply:
Good catch, it is a typo. The correct name should be:
```suggestion
func TestOutputTestCreationWithConfigOverwrite(t *testing.T) {
```